js/src/features/project/components/ProjectCreateForm.tsx

```diff
@@ -18,11 +19,15 @@ export const CreateProjectForm: React.FC<Props> = ({
 }) => {
   const [form] = Form.useForm();
 
-  useFormServerErrors(form, errors, ["name", "description"]);
+  useFormServerErrors(form, errors, ["name", "description", "slug"]);
 
   const onFinish = useCallback(
     (values: any) => {
-      create(values);
+      create({
+        description: values.description,
+        name: values.name,
+        slug: !!values.slug ? convertToSlug(values.slug) : convertToSlug(values.name)
```

Comment:
It is not perfect solution. I imagine the flow in the following way:
1. User is typing the name of project
2. In the same moment automatically filled slug field (like computed field)
3. If user wants to modify slug, he can do this. And if thereafter he wants to change the name, slug should not be changed, cause user already choose slug name. To facilitate the task you can use `touched` property of form field (https://ant.design/components/form/#FieldData)

Reply:
Fix